enterprise/tailnet/pgcoord.go

```diff
@@ -1247,7 +1323,7 @@ type querierWorkKey struct {
 }
 
 type queueKey interface {
-	bKey | tKey | querierWorkKey
+	bKey | tKey | querierWorkKey | hKey
```

Comment:
I think hKey itself can be dropped and the fields directly put into `readyForHandshake` since we don't queue up this kind of work anymore.